src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -162,6 +168,38 @@ public Object instanceMain(final String[] argv) {
     }
 
     /**
+     * Propagate the verbosity level to both the picard and log4j loggers.
+     */
+    protected void setLoggingLevel(final Log.LogLevel VERBOSITY) {
```

Comment:
Could this method be made static and moved into Utils?  I think it makes more sense as a static method since it sets global static state rather than anything specific to this class.